source-checker.pl

```diff
@@ -200,7 +200,16 @@ BEGIN
     if (%patches) {
         # parsing changes
         for my $changes (@changes) {
-            my $diff = diff "$old/$changes", "$dir/$changes";
+            my $diff = "";
+            if (! -e "$old/$changes") {
+                $diff = diff "/dev/null", "$dir/$changes";
+            }
+            elsif (! -e "$dir/$changes") {
```

Comment:
I doubt the new .changes file will ever be missing - as we do run over a glob of changes file in the new package